Formula/google-authenticator-libpam.rb

```diff
@@ -0,0 +1,42 @@
+class GoogleAuthenticatorLibpam < Formula
+  desc "PAM module for two-factor authentication"
+  homepage "https://github.com/google/google-authenticator-libpam"
+  url "https://github.com/google/google-authenticator-libpam/archive/1.04.tar.gz"
+  sha256 "8284cc046be436513d9d4bbb1285017327edbcc32f6f620c47e7e889c4b966ef"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "qrencode" => :recommended
+
+  def install
+    cd "src" do
```

Comment:
Not a big thing but not really worth `cd`ing into a directory to execute an inreplace. Just do:
```ruby
inreplace "src/google-authenticator.c",
          "libqrencode.3.dylib",
          "libqrencode.4.dylib
```